app/models/king.rb

```diff
@@ -1,4 +1,24 @@
 class King < Piece
-	def move
+
```

Comment:
It's probably okay to skip it for this app but in a real world application you would want to actually validate that the co-ordinates that the user wants to move to or valid in the first place.  For example a user could put in a start or end point that was off the board and this method would not detect it.  I'm not saying that this has to be done for this app but just pointing out what would need to be done for a public production app.